bin/download-firefox-artifact

```diff
@@ -1,13 +1,28 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
 
 # Forked from https://github.com/devtools-html/debugger.html/blob/master/bin/download-firefox-artifact
 #
 # This looks for a mozilla-central artifact build as a sibling of the
 # activity-stream tree.  If it's not there, it creates it.  If it is there, it
 # updates it.
 
-ROOT=`dirname $0`
-FIREFOX_PATH="$ROOT/../../mozilla-central"
+# If AS_GIT_BIN_REPO (the git repo from which prepare-mochitests-dev and
+# friends will be executed) isn't set in the environment, just use the repo
+#  we're running from.
+if [ -z ${AS_GIT_BIN_REPO+x} ]; then
```

Comment:
Eh, I suppose if we tried to refactor to share this with `prepare-mochitest-dev`, that shared script would also need its own set of detecting which version to use. I guess this is fine for now.